extensions/wikia/AdEngine3/src/slot/fmr-rotator.js

```diff
@@ -247,18 +248,21 @@ function tryPushNextSlot() {
  * @returns {void}
  */
 export function rotateIncontentBoxad(slotName) {
+	let nextSlot = null;
```

Comment:
It's unclear what is `nextSlot` and `nextSlotName`. Rename? Or maybe (better) some unification since afaik this variable have same value at some point of code execution.